nilearn/interfaces/fmriprep/load_confounds_strategy.py

```diff
@@ -37,6 +37,7 @@
         "motion": "full",
         "n_compcor": "all",
         "compcor": "anat_combined",
+        "global_signal": None,
```

Comment:
Do we want to add this in an example?

Reply:
In fact we cannot create an example with the example dataset in the same tutorial because there's no json file for the confounds. We will have to update the example dataset to do so

Reply:
I think this the 3rd instance where we cannot show things in example because we rely on too old datasets. Probably worth opening an issue to identify what BIDS dataset we have could potentially be updated.

Reply:
Sounds good to me. I did downsampled two fMRIPrep preprocessed dataset for my other projects as test data (ds000228, ds000030).